tests/Integration/classes/CartGetOrderTotalTest.php

```diff
@@ -58,7 +58,7 @@ public static function setUpBeforeClass()
 
```

Comment:
@todo => refactor `DatabaseDump` class to fix the security warning of MySQL 5.7.

Reply:
not needed anymore :+1: